server/services/auth_service.go

```diff
@@ -100,3 +103,29 @@ func GetUserFromSession(c *fiber.Ctx) (*models.User, error) {
 
 	return user, nil
 }
+func InsertUserToSession(c *fiber.Ctx, user *models.User) error {
+	sess, err := GetStore(c)
+	if err != nil {
+		log.Error(fmt.Printf("this is the error %s", err))
+		return fiber.NewError(fiber.StatusInternalServerError)
+	}
+
+	sess.Set("user", user)
+
+	if err = sess.Save(); err != nil {
+		log.Error(fmt.Printf("this is the error %s", err))
+		return fiber.NewError(fiber.StatusInternalServerError)
+	}
+
+	return nil
+}
+func GetUserFromCookiesByJWT(c *fiber.Ctx) (*models.User, error) {
+	cookie := c.Cookies("user")
+
+	if cookie == "" {
+		return nil, fiber.NewError(fiber.StatusUnauthorized)
```

Comment:
```suggestion
		return nil, err
```

Reply:
Return a custom error then, just not the status code. 
return "user not in session"